src/util/git.ts

```diff
@@ -28,6 +55,10 @@ export const getDiffAgainstMaster = async (): Promise<DiffObject> => {
   const hasDiffFromMaster = !!changedFilesString.length
   if (!hasDiffFromMaster) throw new Error(NO_DIFFERENCE)
 
+  const isMasterBranch = await didCheckoutFromMaster()
+  if (typeof isMasterBranch === 'string') console.log(isMasterBranch)
```

Comment:
I thought it was not right to cancel the sending of the submission because of `git reflog` failing to return an output. So, I chose to log a message.